### PR TITLE
scraped 114 plants

### PR DESCRIPTION
**Pull request by olhadanylchenko**

### CODE REVIEW

**Comment by olhadanylchenko** [comment]
@bl00dymarie @karlkristopher please review

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "start": "node ./server",
     "dev": "nodemon ./server",
-    "dev:debug": "DEBUG=green-thumb* npm run dev"
+    "dev:debug": "DEBUG=green-thumb* npm run dev",
+    "scrape": "node ./seed/scraper.js"
```
Comment (bl00dymarie) [inline package.json:9]: :fire:

File: seed/scraper.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+          .querySelectorAll(".table")[1]
+          .querySelector("tbody > tr:nth-child(3) > td:nth-child(2)").innerText;
+      } catch (exception) {}
+      try {
+        plant.soil = document
+          .querySelectorAll(".table")[1]
+          .querySelector("tbody > tr:nth-child(4) > td:nth-child(2)").innerText;
+
+        // plant.watering: document.querySelector("div:nth-child(11) > div.row.row-code > div > table > tbody > tr:nth-child(3) > td:nth-child(2)")
+      } catch (exception) {}
```
Comment (bl00dymarie) [inline seed/scraper.js:169]: Do all the `try { plant.light }`, `try { plant.soil }` etc. need to be present for a plant to be included? or can a plant not have e.g. `soil` information?

File: seed/seeds.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+// });
+
+// Plant.insertMany(plants)
+//   .then((data) => {
+//     console.log("Success");
+//     mongoose.connection.close();
+//   })
+//   .catch((err) => {
+//     console.log(err);
+//   });
```
Comment (bl00dymarie) [inline seed/seeds.js:15]: Why do you include this here?

Comment (olhadanylchenko) [reply seed/seeds.js]: i absolutely dont. deleted!

**Pull request merged**